ci-operator/config/cloud-bulldozer/plow/cloud-bulldozer-plow-master.yaml

```diff
@@ -0,0 +1,18 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.12
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: etcd-perf-test-daily
```

Comment:
You'll need to add also `tag_specification` needed by ci-operator to know what version of OpenShift to install.

Reply:
That's also the reason why the job's rehearsal fails.